pkg/volume-rpc/targetserver/config.go

```diff
@@ -45,13 +45,13 @@ func (csr *CVReplicationDetails) BuildConfigData() map[string]string {
 	// key represents what kind of configurations
 	// value represents corresponding value for that key
 	// TODO: Improve below code by exploring different options
-	key := fmt.Sprintf("  ReplicationFactor")
+	key := "  ReplicationFactor"
 	value := fmt.Sprintf("  ReplicationFactor %d", csr.ReplicationFactor)
 	data[key] = value
-	key = fmt.Sprintf("  ConsistencyFactor")
+	key = "  ConsistencyFactor"
 	value = fmt.Sprintf("  ConsistencyFactor %d", csr.ConsistencyFactor)
 	data[key] = value
-	key = fmt.Sprintf("  Replica %s", csr.ReplicaID)
+	key = string(csr.ReplicaID)
```

Comment:
It should be in the following format
```go
key = "  Replica " + csr.ReplicaID
```
above output should be **"  Replica \<replica ID\>"**(two spaces at starting of a string) but with a changes in PR it will get modified to `<replica ID>`.
Can you please update the above snippet?

Reply:
This is required since we are comparing the above keys over [here](https://github.com/openebs/api/blob/d18d04d8711235a9f217d20542b925f7fe8943f3/pkg/util/fileoperator.go#L74) with equality operator.